src/main/java/com/example/springboot/Application.java

```diff
@@ -1,13 +1,25 @@
 package com.example.springboot;
 
-import org.springframework.boot.SpringApplication;
+import lombok.extern.log4j.Log4j2;
+import org.springframework.boot.ApplicationRunner;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.builder.SpringApplicationBuilder;
+import org.springframework.context.annotation.Bean;
+import org.springframework.core.env.Environment;
 
+@Log4j2
 @SpringBootApplication
 public class Application {
 
   public static void main(String[] args) {
-    SpringApplication.run(Application.class, args);
+    new SpringApplicationBuilder()
+            .profiles("test")
+            .sources(Application.class)
+            .run(args);
   }
 
+  @Bean
+  ApplicationRunner applicationRunner(Environment environment) {
```

Comment:
It's better to move the beans configuration into a separate file and mark it with the annotation `@Configuration`.